src/app/mocks/MswProvider.tsx

```diff
@@ -0,0 +1,40 @@
+'use client';
+
+import { Suspense, use } from 'react';
+
+const mockingEnabledPromise =
+  typeof window !== 'undefined'
+    ? import('./browser').then(async ({ worker }) => {
+        await worker.start({
+          onUnhandledRequest(request, print) {
+            if (request.url.includes('_next')) {
+              return;
+            }
+            print.warning();
+          },
+        });
+      })
+    : Promise.resolve();
```

Comment:
P4: then 내 동작되는 함수가 하나라서 async가 필요없어 보여서 아래와 같이 수정이 가능할 것 같은데 어떠신가요?

```suggestion
const mockingEnabledPromise =
  typeof window !== 'undefined'
    ? import('./browser').then(({ worker }) =>
        worker.start({
          onUnhandledRequest(request, print) {
            if (request.url.includes('_next')) {
              return;
            }
            print.warning();
          },
        }),
      )
    : Promise.resolve();
```

Reply:
아 그렇네요 수정하겠습니다!

Reply:
저는 2번이 괜찮다고 생각한게, 1번으로 갈 시에 msw 말고 따로 사용치않는 axios 도입 및 api 코드를 두 번 작성되야 되는것이 애매해서 2번을 희망합니다!


-> 이 부분에 대해서는 1번으로 진행하더라도 axios를 사용하는 대신, 공통적으로 사용하는 fetch 함수를 구현하고 거기서 캐시 기능을 끌 생각입니다!